packages/react-core/src/components/Divider/Divider.tsx

```diff
@@ -25,13 +25,23 @@ export interface DividerProps extends React.HTMLProps<HTMLElement> {
     xl?: 'insetNone' | 'insetXs' | 'insetSm' | 'insetMd' | 'insetLg' | 'insetXl' | 'inset2xl' | 'inset3xl';
     '2xl'?: 'insetNone' | 'insetXs' | 'insetSm' | 'insetMd' | 'insetLg' | 'insetXl' | 'inset2xl' | 'inset3xl';
   };
+  /** Indicates how the divider will display at various breakpoints. Vertical divider must be in a flex layout. */
+  orientation?: {
+    default?: 'vertical' | 'horizontal';
+    sm?: 'vertical' | 'horizontal';
+    md?: 'vertical' | 'horizontal';
+    lg?: 'vertical' | 'horizontal';
+    xl?: 'vertical' | 'horizontal';
+    '2xl'?: 'vertical' | 'horizontal';
+  };
 }
 
 export const Divider: React.FunctionComponent<DividerProps> = ({
   className,
   component = DividerVariant.hr,
   isVertical = false,
   inset,
+  orientation,
```

Comment:
can we or should we set the default orientation alongside the other default props? Before, `isVertical` defaulted to false.